crates/symbolicator-service/src/caching/memory.rs

```diff
@@ -324,7 +321,10 @@ impl<T: CacheItemRequest> Cacher<T> {
         let name = self.config.name();
         metric!(counter("caches.access") += 1, "cache" => name.as_ref());
 
+        let mut was_memory_hit = true;
         let compute = Box::pin(async {
+            was_memory_hit = false;
```

Comment:
This variable is not used anywhere, is that intentional?

Reply:
lol, good catch!